src/WormholeFees.sol

```diff
@@ -20,6 +20,15 @@ import "./WormholeGUID.sol";
 
 // Calculate fees for a given Wormhole GUID
 interface WormholeFees {
+    /**
+    * @dev Return fee for particular wormhole. It should return 0 for wormholes that are being slow withdrawn. 
+    * note: We define slow withdrawal as wormhole older than x. x has to be enough to finalize flush (not wormhole itself).
```

Comment:
I feel that this is more a clarification for the actual implementation than the interface.
I mean I do not see a case why a whormhole structure would be used if the slow withdraws would pay fee when with a standard bridge wouldn't that happen. Still I think that needs to be left to the implementation.

Reply:
I am not sure if I understand but it's important details b/c bridges assume 0 fees: https://github.com/makerdao/optimism-dai-bridge/pull/59/files#diff-58e6b13b3c3d3118d206273c88eba18926c6a5e88e35c3910e1dd5eb5682a99bR76

Reply:
Yeah ok, but it is actually a thing from the fees implementation. In the future we might have a fees contract that always charges a fee and a bridge that accepts that. Anyway not an important thing.